msu/hooks/items/item_container.nut

```diff
@@ -39,6 +39,43 @@
 		this.m.Actor.getSkills().update();
 	}
 
+	o.getStaminaModifier <- function( _slots = null )
+	{
+		if (_slots == null)
+		{
+			 _slots = clone ::Const.ItemSlot;
+			 delete _slots.None;
+			 delete _slots.COUNT;
+		}
+		else
+		{
+			if (typeof _slots == "integer") _slots = [_slots];
+		}
+
+		local ret = 0;
+
+		foreach (slot in _slots)
+		{
+			local items = this.getAllItemsAtSlot(slot);
+			foreach (item in items)
+			{
+				// Avoid exceptions for items which don't have getStaminaModifier()
+				// This handles cases such as Quivers in ammo slot which don't have
+				// this function in vanilla but do in mods like Legends
+				try
+				{
+					ret += item.getStaminaModifier();
+				}
+				catch (error)
+				{
+					::MSU.Mod.Debug.printError(error + " -- for item: " + item.getID(), "debug");
```

Comment:
Why try catch instead of checking `if ("getStaminaModifier" in item)`

Reply:
Agree with Taro but overall a fan of this commit

Reply:
Because of WeakTableRef shit

Reply:
Will redo this based on MSU.isIn